RELEASE_NOTES.md

```diff
@@ -8,6 +8,81 @@ This document contains release notes for the current major release and all patch
 For prior releases,
 see [PRIOR\_RELEASE\_NOTES.md](PRIOR_RELEASE_NOTES.md).
 
+## 14.0.65 (2020-10-26)
+
+## #New Features
+
+  - The verbose option for the user list API now also includes the display_name attribute.
+
+### Bug Fixes
+  - A bug in our pre-flight validation for Elasticsearch heap_size that resulted in a confusing error rather than a helpful validation warning has been fixed.
+
+  - We now explicitly set TMPDIR during Elasticsearch startup. This should resolve Elasticsearch startup failures for some users who have temporary filesystems mounted with the noexec option and are using SELinux.
+
+  - We now retry in more cases when attempting to create the Elasticsearch index which should prevent upgrade failures for some users.
+
+### Components
+
+ * Updated Components
+
+    - elasticsearch (6.8.1 -> 6.8.12)
+
+## 14.0.58 (2020-9-30)
+
+### Upgrade Notes
+
+  Upgrading to Chef Infra Server 14 will require a reindexing operation for most users. We expect this reindexing operation to take an estimated 2 minutes for each 1000 nodes. However, this estimate can be substantially impacted by your server hardware and the complexity of your Chef data. Generally the size of Elasticsearch indices is observed to be 1/10th of the Postgres database size. To be safe we would recommend that the filesystem containing the /var/opt/opscode directory should have at least 20% free space before upgrade. Learn more about upgrading to Chef Infra Server 14 18.
```

Comment:
require a reindexing operation for users not already using elasticsearch

Reply:
I agree. These release noted are picked up from what we have put on discourse. But I think it might still be ok to get some corrections in.